### PR TITLE
make the gridster support RTL screens 

### PR DESCRIPTION
**Pull request by Shadyelshrief**

### CODE REVIEW

File: projects/angular-gridster2/package.json [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-  "version": "6.0.6",
+  "name": "angular-gridster2-kn",
+  "version": "6.0.10",
   "license": "MIT",
   "peerDependencies": {
     "@angular/common": "^6.0.0",
     "@angular/core": "^6.0.0"
   },
-  "homepage": "https://tiberiuzuld.github.io/angular-gridster2",
+  "homepage": "https://github.com/Shadyelshrief/angular-gridster2",
```
Comment (matpag) [inline projects/angular-gridster2/package.json]: Why you changed the URL home?

File: projects/angular-gridster2/package.json [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
     "url": "https://github.com/tiberiuzuld/angular-gridster2/issues"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/tiberiuzuld/angular-gridster2.git"
+    "url": "https://github.com/Shadyelshrief/angular-gridster2.git"
   },
-  "description": "Angular gridster 2",
+  "publishConfig": {
+    "registry": "http://imore-pn.tradenet.com:8081/repository/kn-npm-hosted/"
```
Comment (matpag) [inline projects/angular-gridster2/package.json]: What is this?

**Comment by matpag** [comment]
By the way, I think a grid should not have the concept of "right" or "left".
If people have a RTL language, they will simply start to put the gridster-items from the right.

I don't think we need this concept in gridster

**Comment by M90K7** [comment]
On RTL Language (like Persian, Arabic ,...) need add the gridster-items from right to left, when add item from right is not zero position of pages (x=0)
Please support RTL option.

**Pull request closed without merging**